banking-stage-ingress-types/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "agave-banking-stage-ingress-types"
+description = "Agave banking stage ingress types"
+documentation = "https://docs.rs/agave-banking-stage-ingress-types"
+version = { workspace = true }
+authors = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+edition = { workspace = true }
+
```

Comment:
These are basically internal types (at least for now); I'm not sure we should publish this crate, wdyt?
i.e. add `publish = false` here

Reply:
Maybe its' fine since it would make it easier for someone to write their own banking stage replacement (good)